server/api/fetch_casinos.go

```diff
@@ -0,0 +1,35 @@
+package api
```

Comment:
Cannot do this, because request handlers depend on websoket session, and session depends on request handlers. If handlers are in a separate package, the GoLang circular dependencies problem aqquires